pkg/util/validation/limits_test.go

```diff
@@ -667,3 +667,73 @@ func TestEnabledByAnyTenant(t *testing.T) {
 
 	require.True(t, EnabledByAnyTenant([]string{"tenant1", "tenant2", "tenant3"}, ov.NativeHistogramsIngestionEnabled))
 }
+
+func TestExtensions(t *testing.T) {
+	t.Cleanup(func() { registeredExtensions = nil })
+
+	// Downstream declares a new extension type and registers it.
+	type testExtensions struct {
+		Foo int `yaml:"foo"`
+	}
+	// By registering, we get a function that provides the extensions for a Limist instance.
```

Comment:
```suggestion
	// By registering, we get a function that provides the extensions for a Limits instance.
```

Reply:
Done in some intermediate commit. Thanks.